x/staking/types/params.go

```diff
@@ -191,3 +204,19 @@ func validateBondDenom(i interface{}) error {
 
 	return nil
 }
+
+func validateMinCommissionRate(i interface{}) error {
+	v, ok := i.(sdk.Dec)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	if v.IsNegative() {
+		return fmt.Errorf("minimum commission rate cannot be negative: %s", v)
+	}
+	if v.GT(sdk.OneDec()) {
+		return fmt.Errorf("minimum commission rate cannot be greater than 100%%")
```

Comment:
```suggestion
		return fmt.Errorf("minimum commission rate cannot be greater than 100%")
```

Reply:
I think it has to be like this

https://programming.guide/go/fmt-printf-percent-sign.html

Reply:
oh im surprised fmt.Errorf doesnt fail lint if theres no extra args